R/dvhydrograph-render.R

```diff
@@ -398,8 +398,8 @@ plotDVHydroLegend <- function(plot_object, startDate, endDate, timezone, initial
   legend_offset <- ifelse(ncol==2, initialOffset+(0.025*leg_lines), initialOffset/2+(0.025*leg_lines))
   legend_offset <- legend_offset * modOffset
 
-  #If the time period is greater than 1 year additional x-axis labels are added so we must move the legend down further
-  legend_offset <- ifelse(as.period(interval(startDate, endDate, tzone = attr(startDate, timezone))) < lubridate::years(1), legend_offset+0.03, legend_offset+0.08)
+  #Add space to legend so that it doesn't overlap with dates.
+  legend_offset <- legend_offset + 0.8
```

Comment:
Does this not need to be dynamic anymore? I think it was dynamic because the extra labels that caused the overlap are only added on plot ranges of 1 year or greater and we didn't want the extra space to be there when it didn't need to be, but if this isn't an issue anymore then I think this looks good!